hooks/pre/35-change-evr-extension-ownership.rb

```diff
@@ -0,0 +1,17 @@
+if local_postgresql?
+  response, _status = execute_command("sudo systemctl is-active postgresql", false, true)
```

Comment:
The installer already runs as root, so no need for sudo
```suggestion
  response, _status = execute_command("systemctl is-active postgresql", false, true)
```

Reply:
Updated..